File: College/Exercises/Introdução à Programação de Computadores/Lista de Exercícios/Lista (Vetor e Matriz)/Exercício11.py
arr = []
multiplos = []

print('Digite 10 números: ')

for i in range(10):
    arr.append(int(input('-> ')))

print(f'Vetor: \n{arr}')

num = int(input('Informe o número desejado: '))

for i in arr:
    if i % num == 0:
        multiplos.append(i)

print(f'Há {len(multiplos)} múltiplos de {num}. São eles:\n{multiplos}')

File: C/Exercises/4ª Lista de Exercícios/Exercício11.c
#include <stdio.h>

int main()
{
    int num, n1, n2, s;

    for (num = 1000; num < 10000; num++)
    {
        n1 = num / 100;
        n2 = num % 100;
        s = n1 + n2;

        if ((s * s) == (num))
        {
            printf("%d\n", num);
        }
    }
}